empire.php

```diff
@@ -10,6 +10,7 @@
  */
 require __DIR__ . '/vendor/autoload.php';
 require __DIR__ . '/Empire/helpers.php';
+require __DIR__ . '/Empire/public.php';
```

Comment:
Not really scoped to this PR, but these files should be included in the [autoload](https://getcomposer.org/doc/04-schema.md#files) configuration.

Reply:
Thank you, Ryan. Done